src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceCrudScenarioTests.ps1

```diff
@@ -22,30 +22,31 @@ function Test-CreateManagedInstance
 {
 	# Setup
 	$rg = Create-ResourceGroupForTest
-	 	
+	$rgName = "mi-wcus-demo-rg" 	
  	$managedInstanceName = Get-ManagedInstanceName
  	$version = "12.0"
  	$managedInstanceLogin = "dummylogin"
  	$managedInstancePassword = "Un53cuRE!"
- 	$subnetId = "/subscriptions/ee5ea899-0791-418f-9270-77cd8273794b/resourceGroups/cl_one/providers/Microsoft.Network/virtualNetworks/cl_initial/subnets/CooL"
+ 	$subnetId = "/subscriptions/a8c9a924-06c0-4bde-9788-e7b1370969e1/resourceGroups/mi-wcus-demo-rg/providers/Microsoft.Network/virtualNetworks/mi-wcus-vnet/subnets/mi-subnet-managed-instances"
```

Comment:
Since MI is in GA now, can you please change this test to be fully automated? This includes creating the virtual network. Check out SQL virtual network firewall rules for an example.